lib/transactionsFilter/TransactionHashesCache.js

```diff
@@ -0,0 +1,49 @@
+class TransactionHashesCache {
+  constructor() {
+    this.transactionHashes = {};
+    this.cacheSize = 10;
+  }
+
```

Comment:
just curious, how is `this.cacheSize` handled in dashd? At what point are unconfirmed transactions deleted from the mempool?